### PR TITLE
2018/09/18/1

### PR DESCRIPTION
**Pull request by arek1-618**
Added improvement to edit description and improvement of translations.
chrome_2018-09-18_11-27-27

### CODE REVIEW

File: modules/Settings/Picklist/actions/SaveAjax.php [hunk 1]
```diff
@@ -93,7 +93,7 @@ public function rename(\App\Request $request)
 		$fieldModel = Settings_Picklist_Field_Model::getInstance($pickListFieldName, $moduleModel);
 		$selectedFieldNonEditablePickListValues = App\Fields\Picklist::getNonEditablePicklistValues($fieldModel->getName());
 		if (isset($selectedFieldNonEditablePickListValues[$id])) {
-			throw new \App\Exceptions\IllegalValue('ERR_NOT_ALLOWED_VALUE');
+			$newValue = $oldValue;
```
Comment (rskrzypczak) [inline modules/Settings/Picklist/actions/SaveAjax.php]: skip permission?

File: layouts/basic/modules/Settings/Picklist/EditView.tpl [hunk 2]
```diff
@@ -45,34 +46,37 @@
...
+						{/if}
+						<div class="form-group row align-items-center">
+							<div class="col-md-3 col-form-label text-right">
+								{\App\Language::translate('LBL_DESCRIPTION',$QUALIFIED_MODULE)}
+							</div>
+							{if empty($PICKLIST_VALUE['description'])}
+								{assign var="DESCRIPTION" value=''}
+							{else}
+								{assign var="DESCRIPTION" value=$PICKLIST_VALUE['description'] }                                    {/if}
+
```
Comment (rskrzypczak) [inline layouts/basic/modules/Settings/Picklist/EditView.tpl]: Where {/if}?
Remove $DESCRIPTION variable

File: layouts/basic/modules/Settings/Picklist/EditView.tpl [hunk 3]
```diff
@@ -10,12 +10,14 @@
 ********************************************************************************/
 -->*}
 {strip}
+	{assign var=OLD_VALUE value=\App\Purifier::encodeHtml($PICKLIST_VALUE['picklistValue'])}
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Settings/Picklist/EditView.tpl]: tpl-Settings-Picklist-EditView

File: modules/Settings/Picklist/actions/SaveAjax.php [hunk 4]
```diff
@@ -93,7 +93,9 @@ public function rename(\App\Request $request)
 		$fieldModel = Settings_Picklist_Field_Model::getInstance($pickListFieldName, $moduleModel);
 		$selectedFieldNonEditablePickListValues = App\Fields\Picklist::getNonEditablePicklistValues($fieldModel->getName());
 		if (isset($selectedFieldNonEditablePickListValues[$id])) {
```
Comment (mariuszkrzaczkowski) [inline modules/Settings/Picklist/actions/SaveAjax.php]: if (isset($selectedFieldNonEditablePickListValues[$id]) && !empty($newValue) ) {

**Pull request merged**